### PR TITLE
Fix bad merge of #943

### PR DESCRIPTION
**Pull request by Ebag333**
Looks like there was a bad merge or overwrite, and parts of #943 got dropped/broken.  This fixes that.

@blitmann I'm submitting this against master as it's half implemented and in a weird state.

The global ammo picker is also broken, because it's not imported, but we have it in the preference window. :(

### CODE REVIEW

File: gui/builtinContextMenus/__init__.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 __all__ = [
```
Comment (blitzmann) [inline gui/builtinContextMenus/__init__.py]: If we're not doing `import *`, this file isn't needed anymore, correct?

Comment (blitzmann) [inline gui/builtinContextMenus/__init__.py]: Just let me know here and I can remove it, gonna merge this now

**Comment by Ebag333** [comment]
We don’t technically need the all anymore. Still should have the init file itself.

**Comment by Ebag333** [comment]
it doesn't hurt to have, and can be useful for tests.

**Pull request merged**